src/useGeolocation.ts

```diff
@@ -8,7 +8,8 @@ export interface GeoLocationSensorState {
   latitude: number,
   longitude: number,
   speed: number,
-  timestamp: number
+  timestamp: number,
+  error: boolean
```

Comment:
Do you think we can store the error here, or `boolean` is enough?

```ts
error?: Error | PositionError,
```

Reply:
Yes, you are right.
What about using `success` and `error` object so it can be used like that
```javascript
const {success, error} = useGeolocation()
```

Reply:
Maybe call it `loading`, for consistency with other libs; and remove the default `null`s:

![image](https://user-images.githubusercontent.com/9773803/52975004-f72be880-33c3-11e9-9d89-55ebba9b9cff.png)

Reply:
```js
const [state, setState] = useState({
  loading: true,
  timestamp: Date.now(),
});
```

Reply:
So it will be something like ?
```javascript
const [state, setState] = useState({
  loading: true,
  timestamp: Date.now(),
  error?: PositionError,
  data?: GeolocationPosition, // or info 
})
```

Reply:
that will be a breaking change ?

Reply:
> that will be a breaking change ?

Actually, maybe better not to add a breaking change and just add `loading` and `error` fields.

Reply:
Ok. that's right.

Reply:
I don't have access to my labtop now, So i will do it tomorrow